oracle/oracle.go

```diff
@@ -166,6 +167,40 @@ func (o *Oracle) Start(ctx context.Context) error {
 	}
 }
 
+// Starts oracle process without a client for running on an Ojo node that runs a
+// price feeder natively.
+func (o *Oracle) StartClientless(
```

Comment:
```suggestion
func (o *Oracle) StartClientless(
```

"Clientless" is kind of a misnomer here, but I can't seem to find a better word for it